ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -1462,19 +1462,21 @@ def install_kra(host, domain_level=None, first_instance=False, raiseonerr=True):
 
 
 def install_ca(host, domain_level=None, first_instance=False,
-               external_ca=False, cert_files=None, raiseonerr=True):
+               external_ca=False, cert_files=None, raiseonerr=True,
+               extra_args=()):
     if domain_level is None:
         domain_level = domainlevel(host)
     check_domain_level(domain_level)
     command = ["ipa-ca-install", "-U", "-p", host.config.dirman_password,
                "-P", 'admin', "-w", host.config.admin_password]
+    command.append(extra_args)
```

Comment:
This is wrong.
```
>>> t = ()
>>> command = ['ls']
>>> command.append(t)
>>> command
['ls', ()]
```

I think you want `command.extend(extra_args)` instead, a la:
```
>>> command = ['ls']
>>> command.extend(())
>>> command
['ls']
```

The call sites where `extra_args=...` is used must changed also, to a list or tuple.
(See comments below).  So before `command.extend(extra_args)`, ensure the type is correct:
```python
if not isinstance(extra_args, (tuple, list)):
    raise TypeError("extra_args must be tuple or list")
```